src/TaskManager.java

```diff
@@ -0,0 +1,60 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskManager {
+
+    public interface ChecksAndUpdates {
+        ArrayList<Task> filter(List<Task> tasks);
+    }
+
+    private final ArrayList<Task> tasks;
+
+    public TaskManager() {
+        tasks = new ArrayList<Task>();
+    }
+
+    public void add(Task task) {
+        this.tasks.add(task);
```

Comment:
Don't use this.

Use this. Only when initialize member of the class in the constructor

Reply:
whAT?